test/tests.js

```diff
@@ -241,8 +241,29 @@ QUnit.test('false secure value', function (assert) {
 	assert.strictEqual(actual, expected, 'false should not modify path in cookie string');
 });
 
+QUnit.test('unofficial attribute', function (assert) {
+	assert.expect(1);
+	assert.strictEqual(Cookies.set('c', 'v', {
+		unofficial: 'anything'
+	}), 'c=v; path=/; unofficial=anything', 'should write the cookie string with unofficial attribute');
+});
+
+QUnit.test('unofficial attribute with boolean value', function (assert) {
+	assert.expect(1);
+	assert.strictEqual(Cookies.set('c', 'v', {
+		unofficial: true
```

Comment:
> So it's only 9-10 bytes less. Do you thing it is worth?

Yes, even 1 byte is relevant because we're talking about GZipped size. This library is used by a few big websites and at scale it can make a difference.